rdmo/accounts/models.py

```diff
@@ -138,6 +138,12 @@ class Role(models.Model):
         help_text=_('The sites for which this user is manager.')
     )
 
+    content_editor = models.ManyToManyField(
```

Comment:
I would just call it `editor`, all the fields are for `Site`.

Reply:
yes `editor` would be best.
Then for the special case where editor has "all Sites" for example call it the `instance_editor` property?
```py
@property
def instance_editor(self):
    return self.editor.all().count() == Site.objects.all().count()
```